src/app/view/components/_common/dot-page-selector/dot-page-selector.component.ts

```diff
@@ -69,11 +96,19 @@ export class DotPageSelectorComponent implements ControlValueAccessor {
      * @memberof DotPageSelectorComponent
      */
     search(param: string): void {
-        this.dotPageSelectorService
-            .getPagesInFolder(param, this.hostIdentifier)
-            .subscribe((pages: DotPageAsset[]) => {
-                this.results = pages;
-            });
+        if (param.length) {
+            this.dotPageSelectorService
+                .search(param)
+                .pipe(take(1))
+                .subscribe((results: DotPageSelectorResults) => {
+                    this.results = results;
+                    this.message = this.getErrorMessage();
+
+                    if (this.shouldAutoFill() && this.isOnlyFullHost(param)) {
+                        this.autoSelectUniqueResult();
+                    }
+                });
+        }
```

Comment:
don't we need a else here?, I mean what happen if I enter something into the input and then erase it